pkg/util/validation/errors.go

```diff
@@ -26,6 +26,15 @@ func (e *genericValidationError) Error() string {
 	return fmt.Sprintf(e.message, e.cause, formatLabelSet(e.series))
 }
 
+// labelValueTooLongError wraps around genericValidationError for customization of Error
+type labelValueTooLongError struct {
+	innerError genericValidationError
```

Comment:
`labelValueTooLongError` doesn't need to wrap `genericValidationError`, and acccess its inner fields. It can just define its own fields directly.